apps/paper/package.json

```
@@ -7,6 +7,7 @@
     "android": "react-native run-android",
     "build:android": "cd android && ./gradlew assembleDebug --no-daemon --console=plain -PreactNativeArchitectures=arm64-v8a",
     "ios": "bundle install && pod install --project-directory=ios && react-native run-ios",
+    "web": "webpack serve --config webpack.config.js",
```

Looks like `webpack.config.js` is the default value, was this line auto-generated from some template or you wrote it yourself?
> Usually, your projects will need to extend this functionality, for this you can create a webpack.config.js file in the root folder and webpack will automatically use it.
https://webpack.js.org/configuration/#root
```suggestion
    "web": "webpack serve",
```